app/controllers/api/v3/facilities_controller.rb

```diff
@@ -30,8 +30,10 @@ def transform_to_response(facility)
   end
 
   def response_process_token
-    {other_facilities_processed_since: processed_until(other_facility_records) || other_facilities_processed_since,
-     resync_token: resync_token}
+    {
+      other_facilities_processed_since: processed_until(other_facility_records) || other_facilities_processed_since,
+      resync_token: current_resync_token
```

Comment:
`resync_token` is a meaningful name already, I think we don't need the `current` suffix unlike the user and facility header.